README.md

```diff
@@ -19,6 +19,48 @@ Modern vision of the mobile application for the Open edX platform from Raccoon G
 
 6. Click the **Run** button.
 
+## Translations
+### Getting translations for the app
+Translations aren't included in the source code of this repository as of [OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html). Therefore, they need to be pulled before testing or publishing to App Store.
+
+To get the latest translations for all languages use the following command:
+```bash
+make pull_translations
```

Comment:
```suggestion
# Note: This will install Python requirements, therefore it is recommended to enable a Python virtual environment
make translation_requirements
make pull_translations
```